### PR TITLE
Feature/luaextensions

### PR DESCRIPTION
**Pull request by alexanderbock**

### CODE REVIEW

File: src/scripting/systemcapabilitiesbinding.cpp [hunk 1]
```diff
@@ -0,0 +1,341 @@
...
+                "cacheSize",
+                &luascripting::general::cacheSize,
+                "",
+                "Returns the cache size."
+            },
+            {
+                "extensions",
+                &luascripting::general::extensions,
+                "",
+                "Returns all supported exteions as comma-separated string."
```
Comment (emiax) [inline src/scripting/systemcapabilitiesbinding.cpp]: Is there a reason to join the extensions to one string? How about returning a lua array-of-strings?

File: src/scripting/scriptengine.cpp [hunk 2]
```diff
@@ -41,12 +41,12 @@
 #include "scriptengine_lua.inl"
 
 namespace {
-    const std::string MainTemplateFilename = "${OPENSPACE_DATA}/web/luascripting/main.hbs";
-    const std::string ScriptingTemplateFilename = "${OPENSPACE_DATA}/web/luascripting/scripting.hbs";
-    const std::string HandlebarsFilename = "${OPENSPACE_DATA}/web/common/handlebars-v4.0.5.js";
-    const std::string JsFilename = "${OPENSPACE_DATA}/web/luascripting/script.js";
-    const std::string BootstrapFilename = "${OPENSPACE_DATA}/web/common/bootstrap.min.css";
-    const std::string CssFilename = "${OPENSPACE_DATA}/web/common/style.css";
+    const char* MainTemplateFilename = "${OPENSPACE_DATA}/web/luascripting/main.hbs";
```
Comment (emiax) [inline src/scripting/scriptengine.cpp]: I'm just curious: Why is const char* preferred rather than const std::string?

Comment (alexanderbock) [inline src/scripting/scriptengine.cpp]: const std::string has some initialization that happens at application startup time, whereas const char* can be directly used.  Probably noone will be able to notice, but i think it is a good habit not to use std::string if not necessary

File: src/scripting/systemcapabilitiesbinding.cpp (hunk 1 above)
Comment (alexanderbock) [inline src/scripting/systemcapabilitiesbinding.cpp]: Good idea;  I'll change that

**Pull request merged**